cypress/e2e/todoActions.cy.ts

```diff
@@ -0,0 +1,106 @@
+describe("App test", () => {
+  beforeEach(() => {
+    cy.visit("localhost:3000");
+  });
+
+  it("adding todo", function () {
+    //when
+    cy.get('[data-testid="mainPage-addTodoBtn"]').click();
+    cy.get('[data-testid="task-input"]').type("TestTodo");
+    cy.get('[data-testid="name-input"]').type("TestUser");
+    cy.get('[data-testid="deadline-input"]').click().type("2022-11-25");
+    cy.get('[data-testid="description-textarea"]')
+      .click()
+      .type("TestDescription");
+    cy.get(".sc-hAZoDl").click();
+    //then
+    cy.get('[data-testid="AddTodo-form-wrapper"]')
+      .last()
+      .contains("Successfully added Todo");
+    cy.get('[data-testid="main-page-todos-wrapper"]')
+      .contains("h2", "TestTodo")
+      .should("exist");
+  });
+  it("toggle todo as complete/uncomplete", () => {
+    //when
+    cy.get('[data-testid="main-page-todos-wrapper"]')
+      .children()
```

Comment:
jest szansa na dodanie jakiegoś atrybutu po drodze, żeby to ciut uprościć? Chodzi o to całe drzewko elementów tu i w poniższych przykładach

Reply:
albo skorzystać z 'variables / aliasses' tak jak w linku w jednym z komentarzy wyżej